client/my-sites/themes/theme-options.js

```diff
@@ -74,6 +77,34 @@ const activateOnJetpack = {
 	)
 };
 
+const deleteConfirmation = ( themeId, siteId ) => {
+	return ( dispatch, getState ) => {
```

Comment:
Walks like an action, talks like an action 😄 Can we move it to `state/themes/actions` please? This kind of, well, action logic should live there rather than here IMO.